client.go

```diff
@@ -292,6 +297,42 @@ func (client *client) RefreshMetadata(topics ...string) error {
 	return client.tryRefreshMetadata(topics, client.conf.Metadata.Retry.Max)
 }
 
+func (client *client) GetOffsetRange(topic string, partitionID int32) (int64, int64, error) {
```

Comment:
In general I find this something that is generally useful. E.g., for monitoring purposes, you'd also want to know the available range. Of course, we could do two requests for this purpose as well.